src/backend/vulkan/src/physical_device.rs

```diff
@@ -889,6 +902,57 @@ impl adapter::PhysicalDevice<Backend> for PhysicalDevice {
             None
         };
 
+        let dedicated_allocation =
```

Comment:
these all should be based on `requested_features` instead of `enabled_extensions`